### DISCUSSION TITLE
Error: [auth/configuration-not] An internal error has occurred. [ CONFIGURATION_NOT_FOUND ]] in Android

### DISCUSSION THREAD

**Discussion by alevelazcode** [invertase/react-native-firebase#5334] [General]
I followed the instructions in the documentation for android but when I tried to enter with the phone number I got the following message "Error: [auth/configuration-not] An internal error has occurred. [ CONFIGURATION_NOT_FOUND ]]", in IOS there was no problem.

**Comment by mikehardy** [comment]
I'm never sure what's going on when there are fundamental install errors like that, so I wrote a script you can read + run that shows exactly how to do an install from start to finish: https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh - I just checked it for android and the new v12 release and it runs as a skeleton for comparison - you might also watch `adb logcat`
